src/qt/bitcoin.cpp

```diff
@@ -466,6 +467,13 @@ int GuiMain(int argc, char* argv[])
     QCoreApplication::setAttribute(Qt::AA_EnableHighDpiScaling);
 #endif
 
+#if (QT_VERSION <= QT_VERSION_CHECK(5, 9, 8)) && defined(Q_OS_MACOS)
```

Comment:
This version check is correct because #136 concerns Qt compiled on macOS with `depends`. Compiling with `depends` builds `bitcoin-qt` with `Qt 5.9.8`. See: [Qt Depends Package](https://github.com/bitcoin-core/gui/blob/master/depends/packages/qt.mk)

Reply:
> Based on the linked Qt bug, this version check appears wrong?

The link to Qt bug removed from the OP. It is possible that version check could be improved, but I do not know for sure from which Qt version widgets are quite usable (I do not even say about fixing all bugs).

The recent Homebrew's Qt versions look decent (but not ideal). And this PR goal is to fix GUI look while building with depends (including gitian binary).